### DISCUSSION TITLE
Support for Python 3.12

### DISCUSSION THREAD

**Discussion by coopercox315** [pythonnet/pythonnet#2248] [Q&A]
Hi, Just wondering when we can expect support for python 3.12 to be added, its in its final beta right now and will be released in coming weeks. I ask as the new GIL changes seem like they could be quite useful for implementations into .NET.
https://docs.python.org/3.12/whatsnew/3.12.html

**Comment by filmor** [comment]
I started work in https://github.com/pythonnet/pythonnet/pull/2249

Doesn't look too bad, there were some deprecations and there is one test failing.
